templates/sovereign-rollup/run-sovereign-setup-predeployed.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
```

Comment:
Given that this is quite similar to [run-sovereign-setup.sh](https://github.com/0xPolygon/kurtosis-cdk/blob/main/templates/sovereign-rollup/run-sovereign-setup.sh), I'm wondering if we could merge the two files? It would be easier to maintain.

Reply:
Yes, let me do it in another PR !